src/scaladoc/scala/tools/nsc/doc/model/MemberLookup.scala

```diff
@@ -100,9 +100,11 @@ trait MemberLookup extends base.MemberLookupBase {
   private def isJDK(sym: Symbol) =
     sym.associatedFile.underlyingSource.map(f => isChildOf(f, (sys.props("java.home")))).getOrElse(false)
 
+  private val sepRegex = if (java.io.File.separator == "\\") "\\\\" else java.io.File.separator
+
```

Comment:
another possibility would be `java.util.regex.Pattern.quote(java.io.File.separator)`

Reply:
Sounds good, didn't know about that